packages/components/tabs/src/tab-group.stories.ts

```diff
@@ -1,276 +1,339 @@
-      <sl-tab selected>This is the first tab a very looong example of the tab</sl-tab>
-      <sl-tab-panel>Contents tab 1 ${createLipsumParagraphs(10)}</sl-tab-panel>
-
-      <sl-tab>This is the second tab</sl-tab>
-      <sl-tab-panel>Contents tab 2 ${createLipsumParagraphs(3)}</sl-tab-panel>
-
-      <sl-tab>This is the third tab</sl-tab>
-      <sl-tab-panel>Contents tab 3 ${createLipsumParagraphs(2)}</sl-tab-panel>
-
-      <sl-tab disabled>This is the fourth tab (disabled)</sl-tab>
-      <sl-tab-panel>Contents tab 4</sl-tab-panel>
-
-      <sl-tab>This is the fifth tab</sl-tab>
-      <sl-tab-panel>Contents tab 5</sl-tab-panel>
-
-      <sl-tab>This is the sixth tab</sl-tab>
-      <sl-tab-panel>Contents tab 6</sl-tab-panel>
-
-      <sl-tab>This is the seventh tab</sl-tab>
-      <sl-tab-panel>Contents tab 7</sl-tab-panel>
-
-      <sl-tab>This is the eighth tab</sl-tab>
-      <sl-tab-panel>Contents tab 8</sl-tab-panel>
-
-      <sl-tab>This is the nineth tab</sl-tab>
-      <sl-tab-panel>Contents tab 9</sl-tab-panel>
-
-      <sl-tab>This is the tenth tab</sl-tab>
-      <sl-tab-panel>Contents tab 10</sl-tab-panel>
-    </sl-tab-group>`
-};
-
-export const StickyTabs: StoryObj = {
-  render: ({ vertical, alignment }) =>
-    html` <style>
-        sl-tab-group::part(container) {
-          position: sticky;
-        }
-
-        sl-tab-group[vertical]::part(container) {
-          inset-block-start: 16px;
-          align-self: start;
-        }
-
-        sl-tab-group::part(container) {
-          inset-block-start: -4px;
-        }
-      </style>
-      <div class="tab-wrapper">
-        <sl-tab-group ?vertical=${vertical} .alignment=${alignment}>
-          <sl-tab selected>This is the first tab</sl-tab>
-          <sl-tab-panel>Contents tab 1 ${createLipsumParagraphs(10)}</sl-tab-panel>
-
-          <sl-tab>This is the second tab</sl-tab>
-          <sl-tab-panel>Contents tab 2 ${createLipsumParagraphs(3)}</sl-tab-panel>
-
-          <sl-tab>This is the third tab</sl-tab>
-          <sl-tab-panel>Contents tab 3 ${createLipsumParagraphs(2)}</sl-tab-panel>
-
-          <sl-tab disabled>This is the fourth tab (disabled)</sl-tab>
-          <sl-tab-panel>Contents tab 4</sl-tab-panel>
-
-          <sl-tab>This is the fifth tab</sl-tab>
-          <sl-tab-panel>Contents tab 5</sl-tab-panel>
-
-          <sl-tab>This is the sixth tab</sl-tab>
-          <sl-tab-panel>Contents tab 6</sl-tab-panel>
-
-          <sl-tab>This is the seventh tab</sl-tab>
-          <sl-tab-panel>Contents tab 7</sl-tab-panel>
-
-          <sl-tab>This is the eighth tab</sl-tab>
-          <sl-tab-panel>Contents tab 8</sl-tab-panel>
-
-          <sl-tab>This is the nineth tab</sl-tab>
-          <sl-tab-panel>Contents tab 9</sl-tab-panel>
-
-          <sl-tab>This is the tenth tab</sl-tab>
-          <sl-tab-panel>Contents tab 10</sl-tab-panel>
-        </sl-tab-group>
-      </div>`
-};
-
-export const VerticalInSmallContainer: StoryObj = {
```

Comment:
So we will not have overflow vertical version of tabs? Like here in vertical version: https://ant.design/components/tabs#tabs-demo-slide

Reply:
I think we _do_ need a vertical overflow

Reply:
Fixed!